core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -139,7 +171,7 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
    * located inside configured local directories and won't
    * be deleted on JVM exit when using the external shuffle service.
    */
-  private def createLocalDirs(conf: SparkConf): Array[File] = {
+  private def createLocalDirs(): Array[File] = {
```

Comment:
Nit: is this change really required? This is original code so why not just leave it as it is?